manifests/ml-training-on-eks/training-modules.yaml

```diff
@@ -1,5 +1,5 @@
 name: training
-path: git::https://github.com/awslabs/autonomous-driving-data-framework.git//modules/ml-training/training-k8s-deployment?ref=release/3.6.0
+path: modules/ml-training/training-k8s-deployment
```

Comment:
We will need to update this to `git::https://github.com/awslabs/autonomous-driving-data-framework.git//modules/ml-training/training-k8s-deployment?ref=release/3.7.0` once it is released.